README.md

```diff
@@ -77,7 +77,7 @@ Install the [pre-requisites](https://github.com/IBM/container-service-getting-st
 * Set the Kubernetes environment to work with your cluster:
 
 ```
-$ bx cs cluster-config <replace_with_your_cluster_name>
+$ ibmcloud cs cluster-config <replace_with_your_cluster_name>
```

Comment:
i don't actually know if this was a clean replace for `ibmcloud` to `bx`, i'd recommend someone try these commands out.

Reply:
I did try the command :)

Reply:
both `bx` and `ibmcloud` works.

Reply:
![screen shot 2018-09-17 at 9 28 21 pm](https://user-images.githubusercontent.com/11567379/45634681-ac2e4200-bac0-11e8-8247-778c2a49a4a8.png) preview of it :)

Reply:
nice! :)